test-e2e/cip/push-golden.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# About: This script automatest the process of pushing golden iamges for e2e tests.
+# Images are loaded from local archives and pushed to the designated staging repo.
+# The script is triggered within e2e.go during test setup.
+#
+# Usage:
+#   ./push-golden.sh repo-root
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+repoRoot=$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)
+archivePath="${repoRoot}/test-e2e/cip/golden-archives"
```

Comment:
Style nit: prefer snakecase (`repo_root`, `archive_path`) for local variables in shell scripts.